modules/table/src/main/java/org/apache/ignite/internal/table/RecordViewImpl.java

```diff
@@ -78,7 +97,7 @@ public RecordViewImpl(TableStorage tbl, RecordMapper<R> mapper) {
     }
 
     /** {@inheritDoc} */
-    @Override public @NotNull IgniteFuture<Collection<R>> getAllAsync(Collection<R> keyRecs) {
+    @Override public @NotNull CompletableFuture<Collection<R>> getAllAsync(Collection<R> keyRecs) {
```

Comment:
Why null is returned but the annotation specifies notnull ?

Reply:
Oh, Gotcha!
Implementaion is incomplete yet, I'll throw an exception.